target/scripts/startup/setup-stack.sh

```diff
@@ -702,9 +612,15 @@ function _setup_dkim_dmarc
 
     # shellcheck disable=SC2016
     sed -i -E 's|^(smtpd_milters =.*)|\1 \$dmarc_milter|g' /etc/postfix/main.cf
+  else
+    sed -i '/.*inet:localhost:8891/d' /etc/postfix/main.cf
```

Comment:
Ah, forgot this in the commit message: our current `main.cf` still has the varibales `dkim_milter`/`dmarc_milter` and postconf will complain about them not being used when `ENABLE_OPENDKIM=0`/`ENABLE_OPENDMARC=0`. These changes (and line 619-623) get rid of the line and the warning.

Reply:
Isn't this just for `dkim_milter` and `dmarc_filter`? Not better to match on those keys for deleting instead?

```suggestion
    sed -i '/^dkim_milter/d' /etc/postfix/main.cf
```